src/components/memes/MemesDetails.jsx

```diff
@@ -0,0 +1,44 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { fetchingMemesInfo } from '../../redux/memeSlice';
+import classes from './MemesDetail.module.css';
+
+const MemesDetails = () => {
+  const { memesId } = useParams();
+  const dispatch = useDispatch();
+  const { memesData } = useSelector((state) => state.memess);
+  useEffect(() => {
+    dispatch(fetchingMemesInfo());
+  }, [dispatch, memesId]);
+
+  const [filterMemes] = memesData.filter((data) => data.id === memesId);
+  if (!filterMemes || memesData.isLoading) {
+    return <h1>Loading...</h1>;
+  }
+  return (
+    <div className={classes.memes__card}>
+      <h1 className={classes.name__item}>
+        Name:
+        {filterMemes.name}
+      </h1>
+      <img src={filterMemes.url} alt="" className={classes.memes__image} />
+      <div className={classes.memes__parametrs}>
+        <p>
+          Width:
+          {filterMemes.width}
+        </p>
+        <p>
+          Height:
+          {filterMemes.height}
+        </p>
+        <p>
+          Box Count:
+          {filterMemes.box_count}
+        </p>
+      </div>
```

Comment:
[OPTIONAL] I think it would be best if you made the details tubular as in the design and make them fill 100% of the mobile view width.

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/ce4f496c-5e6e-4bbf-9fd4-7af15447ea2b)

Design:

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/00a6e853-db86-4cef-8e25-0dd7684103f2)

Also, it would be nice if you made the app more responsive. There are white spaces around the theme in mobile view making the app look unfinished. It looks better if the content fills 100% of the view width and height.

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/7d262d7d-83e3-40f9-a408-30022b5e57a9)